### PR TITLE
Gitbook deployment changes

### PR DESCRIPTION
**Pull request by soph-g**
Includes instructions and setup to enable GitBook deployment to Heroku. This includes:

- An app.js file to serve the built book
- a deployment script that will build, commit and push the book

Once these changes are merged in, the book will need to be build on master.

### CODE REVIEW

**Comment by maryrosecook** [comment]
Looks good to me.  Thanks so much for doing this!

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+rm ./_book/deploy.sh
+
+echo "Committing build to git"
+echo "**************************"
+
+git add ./_book
+
+git commit -m "automated build of gitbook for deployment"
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
```
Comment (dearshrewdwit) [inline deploy.sh]: Interesting! What's the intention here?

**Comment by dearshrewdwit** [comment]
Looks fab. 🍰

File: deploy.sh (hunk 1 above)
Comment (soph-g) [reply deploy.sh]: Very late replying to this, but the intention was to provide a simple way to build the book and make sure it is pushed up to the current branch. Didn't want to default to master. Think you proposed that I look into writing a script to simplify the deployment steps, was this what you had in mind?

Comment (soph-g) [reply deploy.sh]: I'm going to go ahead and merge in the changes as it's been approved - can always open a small PR to remove or alter the scripts if needed.

**Pull request merged**